source/manual/review-apps.html.md

```diff
@@ -17,6 +17,8 @@ These are 2 examples of GOV.UK apps that have review apps enabled:
 
```

Comment:
On line 16, it says there are "2 examples". Now that there are more, it would be good to just say something more general.